Dockerfile.Windows

```diff
@@ -1,4 +1,7 @@
-FROM mcr.microsoft.com/windows/servercore:1809 as core
+ARG BASE_IMAGE
+ARG BASE_IMAGE_TAG
+
+FROM mcr.microsoft.com/windows/${BASE_IMAGE}:${BASE_IMAGE_TAG} as core
 
 FROM mcr.microsoft.com/windows/nanoserver:1809
```

Comment:
Does this base image need to be configurable?

Reply:
this is kind of workaround, currently we have to use nanoserver due to socket issue. In the future, we need to find a solution because microsoft will stop supporting nanoserver.